backend/src/test/java/com/daedan/festabook/lostitem/domain/LostItemTest.java

```diff
@@ -0,0 +1,108 @@
+package com.daedan.festabook.lostitem.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.daedan.festabook.global.exception.BusinessException;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.http.HttpStatus;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class LostItemTest {
+
+    @Nested
+    class validateImageUrl {
+
+        @Test
+        void 성공() {
+            assertThatCode(() -> LostItemFixture.createWithImageUrl("https://www.test.com/image.png"))
+                    .doesNotThrowAnyException();
+        }
+
```

Comment:
코드가 가로로 너무 길어요
변수로 나누어 한 번 쉬어가고, 설명을 표현하면 어떨까요?
```suggestion
        @Test
        void 성공() {
            String validImageUrl = "https://www.test.com/image.png";
            assertThatCode(() -> LostItemFixture.createWithImageUrl(validImageUrl))
                    .doesNotThrowAnyException();
        }

```

Reply:
저도 길다고 느껴져요. 제가 처음에 작성했을 때는 null일 때, 비어있을 때 둘다 검증을 따로따로 했는데 이 방식은 어떠신가요?